### PR TITLE
Borrowbook class completed

### PR DESCRIPTION
**Pull request by iamharisali**

### CODE REVIEW

**Comment by Lakshansm90** [comment]
Hi @iamharisali,
There's a Conflicting file in your work, could you please resolve it and create a pull request.
Thank you,
Lakshan

File: library/entities/Book.java [hunk 1]
```diff
@@ -37,8 +37,8 @@ public Integer getId() {
 		return iD;
 	}
 
-	public String getTitle() {
-		return tItLe;
+	public String gettitle() {
+		return title;
 	}
```
Comment (BLakshmiPrasanna) [inline library/entities/Book.java]: Hi @iamharisali  it should be getTitle please check once again

File: library/borrowbook/BorrowBookControl.java [hunk 2]
```diff
@@ -9,10 +9,10 @@
 
 public class BorrowBookControl {
 	
-	private BorrowBookUI uI;
+	private BorrowBookUI ui;
```
Comment (BLakshmiPrasanna) [inline library/borrowbook/BorrowBookControl.java:12]: Hi Haris in BorrowBookUI it should be BorrowBookUi

File: library/borrowbook/BorrowBookControl.java [hunk 3]
```diff
@@ -9,10 +9,10 @@
 
 public class BorrowBookControl {
 	
-	private BorrowBookUI uI;
+	private BorrowBookUI ui;
 	
 	private Library library;
-	private Member mEmBeR;
+	private Member ui;
```
Comment (BLakshmiPrasanna) [inline library/borrowbook/BorrowBookControl.java]: Hi, please check the variable ui it is declared twice  for borrowBookUi and  Member

File: library/borrowbook/BorrowBookControl.java [hunk 4]
```diff
@@ -31,8 +31,8 @@ public void SeT_Ui(BorrowBookUI Ui) {
 		if (!sTaTe.equals(CONTROL_STATE.INITIALISED)) 
 			throw new RuntimeException("BorrowBookControl: cannot call setUI except in INITIALISED state");
 			
-		this.uI = Ui;
-		Ui.SeT_StAtE(BorrowBookUI.uI_STaTe.READY);
+		this.ui = Ui;
+		Ui.SeT_StAtE(BorrowBookUI.ui_STaTe.READY);
 		sTaTe = CONTROL_STATE.READY;
```
Comment (BLakshmiPrasanna) [inline library/borrowbook/BorrowBookControl.java]: Hi haaris ui_sTaTe should be uiState

File: library/borrowbook/BorrowBookControl.java [hunk 5]
```diff
@@ -72,28 +72,28 @@ public void ScAnNeD(int bOoKiD) {
...
 		else {
 			ui.DiSpLaY("\nFinal Borrowing List");
-			for (Book bOoK : pEnDiNg_LiSt) 
+			for (Book bOoK : pendingList) 
 				ui.DiSpLaY(bOoK.toString());
 			
-			cOmPlEtEd_LiSt = new ArrayList<Loan>();
-			ui.SeT_StAtE(BorrowBookUI.ui_STaTe.FINALISING);
+				completedList = new ArrayList<Loan>();
+			ui.SeT_StAtE(BorrowBookUI.uI_STaTe.FINALISING);
```
Comment (BLakshmiPrasanna) [inline library/borrowbook/BorrowBookControl.java]: Please check it should be uiState

File: library/borrowbook/BorrowBookControl.java [hunk 6]
```diff
@@ -27,12 +27,12 @@ public BorrowBookControl() {
...
 
-	public void SeT_Ui(BorrowBookUI Ui) {
+	public void setUi(BorrowBookUI ui) {
 		if (!state.equals(ControlState.INITIALISED)) 
 			throw new RuntimeException("BorrowBookControl: cannot call setUI except in INITIALISED state");
 			
-		this.ui = Ui;
-		Ui.SeT_StAtE(BorrowBookUI.uI_STaTe.READY);
+		this.ui = ui;
+		ui.setState(BorrowBookUI.UiState.READY);
```
Comment (BLakshmiPrasanna) [inline library/borrowbook/BorrowBookControl.java:35]: Hi Haris please check  it should be uiState

File: library/entities/Library.java [hunk 7]
```diff
@@ -89,8 +89,8 @@ public int gEt_BoOkId() {
 	}
 	
 	
-	public int gEt_MeMbEr_Id() {
-		return mEmBeR_Id;
+	public int getMember_Id() {
```
Comment (BLakshmiPrasanna) [inline library/entities/Library.java]: please remove under Score for getMemberId

**Comment by BLakshmiPrasanna** [comment]
Hi @Lakshansm90  there are few defects found please notify the author and clear them asap

File: library/borrowbook/BorrowBookControl.java [hunk 8]
```diff
@@ -68,7 +68,7 @@ public void scanned(int bookId) {
 			ui.display("Invalid bookId");
 			return;
 		}
-		if (!book.isAvailable()) {
+		if (!book.iS_AvAiLaBlE()) {
```
Comment (BLakshmiPrasanna) [inline library/borrowbook/BorrowBookControl.java]: Hi @Lakshansm90  please notify Haris that method name should be  isAvailable

**Comment by BLakshmiPrasanna** [comment]
Hi @Lakshansm90 Conflicts are cleared and looks good to merge please notify @iamharisali

**Comment by Lakshansm90** [comment]
Hi @iamharisali ,
Could you please resolve the conflicts and let me know.
Thanks
Lakshan

**Comment by BLakshmiPrasanna** [comment]
Hi @Lakshansm90 please notify harris to merge the pull request

**Pull request merged**